README.md

```diff
@@ -1,47 +1,10 @@
-# MLonCode research playground [![PyPI](https://img.shields.io/pypi/v/sourced-ml.svg)](https://pypi.python.org/pypi/sourced-ml) [![Build Status](https://travis-ci.org/src-d/ml.svg)](https://travis-ci.org/src-d/ml) [![Docker Build Status](https://img.shields.io/docker/build/srcd/ml.svg)](https://hub.docker.com/r/srcd/ml) [![codecov](https://codecov.io/github/src-d/ml/coverage.svg)](https://codecov.io/gh/src-d/ml)
+# MLonCode Core Library
+ [![Build Status](https://travis-ci.org/src-d/ml-core.svg)](https://travis-ci.org/src-d/ml-core)
+ [![codecov](https://codecov.io/github/src-d/ml-core/coverage.svg)](https://codecov.io/gh/src-d/ml-core)
+ [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
```

Comment:
> flake8, and if we want to switch to black, we will have to reformat all our ML code, which is a bit of work

Black is not a linter. Instead it is a tool that automatically reformats your code to make the diffs the smallest possible.

 I am currently using black on top of pylint and flake8 and there is no conflict between them. It complies with PEP8 and all the best coding practices we are currently using at source{d}.

The cool thing about it is that once it is configured, just by running `black .` you ensure your code will pass almost all the linter's checks (for example, it doesn't change the order of imports)
and that your commits will have the smallest diffs possible. In practice, black applies PEP8 automatically, and chooses the line breaks so the code is easier to read.

As this is a tool that changes the code instead of just suggesting changes, it is possible that you may find it a bit "invasive". If that is the case and you don't want to include it in the repo by default, I will as for permission to use it on my own before making the commits. 

I'm asking you this because after some months using it, it really helped me write code faster, and I got used to read code with black style, so I will actually perform better if you let me use it.

Reply:
It's not that I don't want it. I don't see how it changes our code. The plan is to not add anything potentially nuking all the lines now. Once you PR the black-ed code, I will see how it works and decide on the next steps.

As I said, it is either "all or none" decision. Either all ML projects use it, or none of them. The transition may require some work.

Reply:
Sure, I have not "blacked" any line of code, yet. I'm waiting for the Pr so you can see the changes.

In case you decide to make the transition, I can take care of it. I already transitioned most of my GH repos, and it's suprisingly easy.